source/_topics/splitting_configuration.markdown

```diff
@@ -377,7 +393,7 @@ automation: !include_dir_merge_list automation/
   action:
     service: light.turn_on
     entity_id: light.entryway
-    
+
```

Comment:
May as well add line breaks to the other examples while you're in here making changes. :wink: